storage/storage.go

```diff
@@ -43,8 +43,17 @@ type LookupOptions struct {
 	// LatestAnchor only. If set, it will ignore the time boundaries provided and
 	// just use the last available anchor.
 	LatestAnchor bool
+
+	// FilterOptions, if provided, represent the specifications for the filtering to be executed.
+	FilterOptions *FilteringOptions
 }
 
+// FilteringOptions represent the storage level specifications for the filtering to be executed.
+// Operation below refers to the filter function being applied (eg: "latest"), Field refers to the position of the graph clause it
+// will be applied to ("subject", "predicate" or "object") and Value, when specified, contains the second argument of the filter
```

Comment:
super duper nit: missing oxford comma
```suggestion
// will be applied to ("subject", "predicate", or "object") and Value, when specified, contains the second argument of the filter
```

Reply:
Done, thank you Thiago. 👍